src/SQLite.cs

```diff
@@ -822,6 +1091,9 @@ public CreateTableResult CreateTable (Type ty, CreateFlags createFlags = CreateF
 		/// <returns>
 		/// Whether the table was created or migrated for each type.
 		/// </returns>
+#if NET
```

Comment:
should this be `NET8_0_OR_GREATER` ?

Reply:
Ha, force of habit from CsWinRT ahah, fixed in https://github.com/praeclarum/sqlite-net/pull/1222/commits/5985c0228e2c670ebb405c65235fdc85a967c7c7, thank you!